pkg/kubelet/container/helpers.go

```diff
@@ -117,6 +117,22 @@ func HashContainer(container *v1.Container) uint64 {
 	return uint64(hash.Sum32())
```

Comment:
It's still called by the `getStableKey` function, I was worried that just name and image wouldn't uniquely identify as container , but double checking the context I think it's ok, because `pod.UUID` is also used in `getStableKey` so we just need to keep a hash function.